streamlit_template/app.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+import streamlit as st
+
+from toolkit.queries import (QUERY_ENTITY_DISTRIBUTION, QUERY_PROJECT_DOWNLOADS,
```

Comment:
I would recommend adding a pre commit lint for black and sort (Examples in the SynPy)